test/unit/org/apache/cassandra/cql3/validation/operations/SelectTest.java

```diff
@@ -3191,4 +3191,20 @@ public void testCreatingUDFWithSameNameAsBuiltin_FullyQualifiedFunctionNameWorks
         execute("INSERT INTO %s (k1, k2) VALUES (uuid(), 'k2')");
         assertRowCount(execute("SELECT system.token(k1, k2) FROM %s"), 1);
     }
+
+    @Test
+    public void testLikeQueryWithoutFiltering() throws Throwable
+    {
+        createTable("CREATE TABLE %s (k1 uuid, k2 text, PRIMARY KEY (k1, k2))");
+        execute("INSERT INTO %s (k1, k2) VALUES(uuid(), 'John Doe')");
+        assertRows(execute("SELECT k2 from %s where k2 like '%%Doe'"), row("John Doe"));
+    }
+
+    @Test
+    public void testLikeQueryWithFiltering() throws Throwable
+    {
+        createTable("CREATE TABLE %s (k1 uuid, k2 text, PRIMARY KEY (k1))");
+        execute("INSERT INTO %s (k1, k2) VALUES(uuid(), 'John Doe')");
```

Comment:
To ensure that filtering is properly working we should also have some rows that are rejected and that some of them have a null value for the column

Reply:
For completion I would add tests for all `LIKE`flavors: `LIKE_PREFIX, LIKE_SUFFIX, LIKE_CONTAINS, LIKE_MATCHES`

Reply:
@bereng , Thanks added.